### DISCUSSION TITLE
Datatable that supports controls in both footer and header

### DISCUSSION THREAD

**Discussion by Leftium** [vincjo/datatables#84] [Show and tell]
![image](https://github.com/vincjo/datatables/assets/381217/7dc82946-92b7-4d87-ab7a-0c13473c7efe)

I modified @vincjo's awesome datatable:
- Supported in header, footer, or *both*: search/rowsPerPage/rowCount/pagination
- Refactored out `height` and `class:container` in favor of automatic flexbox layout.
- Modified code: https://github.com/Leftium/bangtastic/blob/c6d5d6363b0b17e0dada08f96ba91517d1b2d593/src/lib/components/Datatable.svelte#L15-L108
- Original code: https://github.com/vincjo/datatables/blob/be9bcedb31e3399e4fb27987adfbf1f9134366e5/src/lib/local/Datatable.svelte#L16-L76

**Comment by vincjo** [comment]
That's great, thx for sharing.
Flexbox layout will be added into the next major version
